pkg/util/maps.go

```diff
@@ -0,0 +1,96 @@
+// Copyright 2020 Red Hat, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package util
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+)
+
+const (
+	keyPairSeparator = "="
+	pairSeparator    = ","
+)
+
+// MapContainsMap returns true only if source contains expected map
+func MapContainsMap(source, expected map[string]string) bool {
+	if len(source) == 0 && len(expected) == 0 {
+		return false
+	}
+	if len(source) > 0 && len(expected) == 0 {
+		return false
+	}
+	if len(source) == 0 && len(expected) > 0 {
+		return false
+	}
```

Comment:
Just for better readability/less exit points of a function, can we do something like:
```go
if len(source) == 0 && len(expected) == 0 ||
   len(source) > 0 && len(expected) == 0 ||
   len(source) == 0 && len(expected) > 0 {
   return false
}
```

Reply:
I have mixed feelings about this. Not sure if it has a better readability.

Reply:
Actually when I think about it right now this is the same as writing:

```go
if len(source) == 0 || len(expected) == 0 {
   return false
}
```
:smiley:

Reply:
LOL
yeah, I over thought this.